src/plugins_types/union.c

```diff
@@ -150,6 +150,31 @@ lyb_parse_union(const void *lyb_data, size_t lyb_data_len, uint32_t *type_idx, c
     }
 }
 
+static void
+union_lref_error_rewrite(const struct lyd_node *ctx_node, struct ly_err_item *err, struct lysc_type *type, const void *value, size_t value_len)
```

Comment:
I would prefer a bit different name although that is a small issue. Maybe `union_update_lref_err`. But the doxygen is missing and required.

Reply:
done